### ISSUE TITLE
Getting this error during logout

### ISSUE THREAD

**Issue by SmartcloudZensys** [joforce/JoforceCRM#92]
Getting this error during logout from the crm.

 {"success":false,"error":{"code":"result is not an object","message":"result is not an object"}}

**Comment by dfanaei** [comment]
Yes, Same Here! 
` {"success":false,"error":{"code":"result is not an object","message":"result is not an object"}}`

**Comment by smackrajue** [comment]
Here is the fix

In the file 'modules/Users/actions/Logout.php', replace the function 'logoutRoundcube()' by the following code
```
private function logoutRoundcube()
    {
        global $site_URL, $adb ,$application_unique_key;;
        require_once 'modules/EmailPlus/RoundcubeLogin.class.php';
        $url = $site_URL . 'modules/EmailPlus/roundcube/';
        $account = EmailPlus_Autologin_Model::getAutologinUsers();
        $checkUserId = $adb->pquery('select jo_user_id from users where username = ?', array($account['email']));
        $IsUpdated = $adb->query_result($checkUserId, 0, 'jo_user_id');
        if (empty($IsUpdated)) {
            $adb->pquery('update users set jo_user_id = ? where username = ?', array($current_user->id, $account['email']));
        }
        $hash_key = md5($application_unique_key . date('dFYaG'));
        if ($account) {
            $url .= '?server=' . $account['name'];
            $url .= '&port=' . $account['port'];
            $url .= '&jo_token=' . $hash_key;
            $rcl = new RoundcubeLogin($url, false);
            if ($rcl->isLoggedIn()) {
                $rcl->logout();
            }
        }
    }
```
This issue will be available from next V 2.0 releases
